### PR TITLE
Update rsync command used to fetch logs

### PR DESCRIPTION
**Pull request by justafish**
## Summary

**[Log Files on Pantheon](https://pantheon.io/docs/logs#automate-downloading-logs)** - Although this script is currently fetching all the app container IPs, it then rsyncs from the domain instead of the IPs that dig found. This results in a random assortment of container logs being downloaded and placed into folders that are incorrectly labelled.

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by justafish** [comment]
I've just realised this is the same for the database section too. Will update this PR.

File: source/content/logs.md [hunk 1]
```diff
@@ -136,12 +136,14 @@ SITE_UUID=xxxxxxxxxxx
...
 for app_server in `dig +short appserver.$ENV.$SITE_UUID.drush.in`;
 do
-  rsync -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' $ENV.$SITE_UUID@appserver.$ENV.$SITE_UUID.drush.in:logs/* app_server_$app_server
+  rsync -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' $ENV.$SITE_UUID@$app_server:logs/* app_server_$app_server
 done
 
 # Include MySQL logs
-db_server=`dig dbserver.$ENV.$SITE_UUID.drush.in +short`
-rsync -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' $ENV.$SITE_UUID@dbserver.$ENV.$SITE_UUID.drush.in:logs db_server_$db_server
+for db_server in `dig +short dbserver.$ENV.$SITE_UUID.drush.in`;
```
Comment (deviantintegral) [inline source/content/logs.md:143]: Should we specify `-4` here to match the rsync command below?

**Comment by EdwardAngert** [comment]
thanks for the changes you're offering here, @justafish !
I changed this from `master` to a branch that we can add changes to, and I'll merge this PR and open a new one from that branch

**Comment by justafish** [comment]
@EdwardAngert you can ask the author to check the "Allow edits and access to secrets by maintainers" box (which I forgot to do here. oops :smile: )

**Comment by alexfornuto** [comment]
Allow edits ++, but we also want our CI tests to run, which only kick off for branches from our repo. It's an imperfect solution for an imperfect world.

**Pull request merged**